app/javascript/modules/listingDetails/ListingDetailsFeatures.tsx

```diff
@@ -23,7 +23,14 @@ interface FeatureItemProps {
 const FeatureItem = ({ content, title }: FeatureItemProps) => {
   if (!content) return <></>
 
-  return <Description term={title} description={stripMostTags(content)} markdown={true} />
+  return (
+    <Description
+      term={title}
+      description={stripMostTags(content)}
+      markdown={true}
+      dtClassName="translate"
+    />
+  )
```

Comment:
I think we may want to add a boolean to this function to flag whether a feature item should be machine translated. Some of fields may. We can probably leave out `Neighborhood` and `Year Built`.